pkg/antctl/command_definition_test.go

```diff
@@ -26,9 +26,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+type Foobar struct {
```

Comment:
Since you have defined this struct, I would suggest to replace all exist literal structs by this.

Reply:
Good idea. I have updated the code.